content/docs/dataform-web/how-to-guides/environments-example.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Environments example project
+---
+
+<div className="bp3-callout bp3-icon-info-sign" markdown="1">
+  Environments require <code>@dataform/core >= 1.4.9</code>.
+</div>
+
+## Introduction
+
+This guide will show you how to configure a project with two environments. One environment will be staging, and the other will be production. Typical of this paradigm, staging will be upstream of production, allowing committed code to be tested before being released.
+
+## Creating a project with environments
+
+- Create an empty remote repository on a git provider, for example on [GitHub](https://github.com/new). The repository must be empty (don't create a README file). Alternatively if adding environments to a project that is already linked to a remote git provider, skip to adding the environments file.
+
+- Go on the [dataform web app](https://app.dataform.co/).
+
+  - Create a new project.
+
+  - From settings, click `Migrate project to git provider`.
+
+  - Follow the steps indicated, linking the empty remote repository to the project.
+
+- Change to the development branch in the dataform app. Add the following files:
+
+  - One of our template `sqlx` files, or anything that will show up in the run logs.
+
+  - A schedule.
+
+  - An environments file with the contents:
+
+```json
```

Comment:
IMO the most common use case for environments, and the example we should show should be using commitSHAs not branches (using branches for releases is a nightmare).

I think there should probably be two examples here:

Example 1 - Creating a staging environment
Example 2 - Promoting releases from staging to production

Reply:
I've removed a lot of the process description, and added a warning about branches instead.